ui-modules/blueprint-composer/app/components/catalog-selector/catalog-selector.directive.js

```diff
@@ -127,15 +135,15 @@ function controller($scope, $element, $q, $uibModal, $log, $templateCache, palet
     $scope.getPlaceHolder = function () {
         return 'Search';
     };
-
+    
     $scope.isLoading = true;
 
     $scope.$watch('search', () => {
         $scope.freeFormTile = {
             symbolicName: $scope.search,
             name: $scope.search,
             displayName: $scope.search,
-            supertypes: [$scope.family.superType]
+            supertypes: ($scope.family ? [ $scope.family.superType ] : []),
```

Comment:
`scope.family` is marked as required in the directive specification, you shouldn't need that

Reply:
This directive wasn't designed to support `null` family. Something is wrong is that's the case, I don't see any reason why we would use it without a family

Reply:
pretty sure i added it because of an NPE here.  not sure how/why but this check won't hurt.